anclafx/provide.go

```diff
@@ -15,8 +15,9 @@ func Provide() fx.Option {
 		Module,
 		fx.Invoke(chrysom.ProvideStartListenerClient),
 		fx.Provide(
-			ancla.ProvideListener,
 			ancla.ProvideService,
+			ancla.ProvideListener,
+			ancla.ProvideDefaultListenerWatchers,
```

Comment:
I forgot to add the default watchers to the `anclafx`.   `webhook list size` metric is currently the only default watcher